### DISCUSSION TITLE
ui chip component - alignment of icon and text

### DISCUSSION THREAD

**Discussion by jenniferarnesen** [dhis2/notes#236] [Q&A] [answered]
I'm converting to ui6 icons in the dashboard app. If I don't add any positioning of the icon, then it is not aligned with the text on the chip:

![image](https://user-images.githubusercontent.com/6113918/108681088-f5cfb200-74ee-11eb-876e-f4d9ec8f0a58.png)

I guess I just need to confirm whether the chip should be responsible for the alignment, or if the app needs to take care of it. Thanks :)

**Comment by mediremi** [comment]
The `Chip` component [applies `display: inline-flex; align-items: center` styles](https://github.com/dhis2/ui/blob/a9f4b87ccabcb7ccd8ed5f9c6e8122a179c9e7ad/packages/core/src/Chip/Chip.js#L51-L52) to its container, which suggests that it is responsible for icon alignment. 

Imo this misalignment is a bug.

**Reply by jenniferarnesen** [reply]
ok, I'll report it to Jira

**Comment by jenniferarnesen** [comment, marked as answer]
https://jira.dhis2.org/browse/LIBS-131
